bot/exts/info/help.py

```diff
@@ -173,8 +173,11 @@ async def command_formatting(self, command: Command) -> Embed:
         if aliases:
             command_details += f"**Can also use:** {aliases}\n\n"
 
-        # check if the user is allowed to run this command
-        if not await command.can_run(self.context):
+        # when command is disabled, show message about it
+        # otherwise check if the user is allowed to run this command
+        if not command.enabled:
+            command_details += "***This command is disabled.***\n\n"
+        elif not await command.can_run(self.context):
```

Comment:
Turns out this was originally broken cause `can_run` will propagate exceptions from check failures. Therefore, simply checking the return value is inadequate. The `CommandError` must be caught.